### DISCUSSION TITLE
Run build jobs in workflows on online sandboxes (Support AppSource App dependencies)

### DISCUSSION THREAD

**Discussion by freddydk** [microsoft/AL-Go#113] [Ideas]
For apps having dependencies on AppSource apps, it could be very nice to add support for running workflows using online environments. 
Run-AlPipeline already supports running full pipelines on online sandbox environments

The Tenant used for spinning up the temporary build environment could be the customer tenant, the partner tenant or maybe even a 3rd (temporary) tenant used for build purposes only.

**Comment by HazardMK** [comment]
Is there any documentation how to set it up? How to run CI/CD on partner tenant?
